src/main/java/com/powsybl/openloadflow/network/extensions/iidm/AsymmetricalBranchConnector.java

```diff
@@ -0,0 +1,57 @@
+/**
+ * Copyright (c) 2023, Jean-Baptiste Heyberger <jbheyberger at gmail.com>
+ * Copyright (c) 2023, Geoffroy Jamgotchian <geoffroy.jamgotchian at gmail.com>
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+package com.powsybl.openloadflow.network.extensions.iidm;
+
+/**
+ * @author Jean-Baptiste Heyberger <jbheyberger at gmail.com>
+ */
+public class AsymmetricalBranchConnector {
```

Comment:
I think this object is not mature enough to go in powsybl-core. The extension `LineAsymmetrical` should stay here too because it depends on this connector.

Reply:
How do you know the type of a bus "wye" or "delta?
Then, it is unexpected that `isPositiveSequenceAsCurrent` is an input data. Same remark for `isFortescueRepresentation`.